modules/smart-agent_systemd-services/conf/00-systemd_services.yaml

```diff
@@ -12,4 +12,4 @@ signals:
 rules:
```

Comment:
I missed this in the previous PR and this is the scope of the current one but please could you increment the file id like `01-substate.yaml` ? (I think using `systemd` is repetitive in a systemd module but the only important thing is to keep `00` for heatbeat ;) )

Reply:
done